src/test/features/login.feature

```diff
@@ -1,9 +1,9 @@
 Feature: Login
 
-Background: 
- Given User passes the authorization
+        Background:
+            Given User passes the authorization
 
- Scenario: Login to Core
-        Given User is in landing page
-        When User logins as a "admin"
-        Then User is in Dashboard page
\ No newline at end of file
+        Scenario: Login to Core
+            Given User is in landing page
+             When User logins as a "admin"
+             Then User is in Dashboard page
```

Comment:
Why do you have 2 separate step definitions to do the same job?
`User is in landing page` and `User is in Dashboard page` can be handled by a single step definition.

Reply:
This feature file related to @YahyoMahmudov

Reply:
It is not my. login.fuature file written by @YahyoMahmudov

Reply:
But you can still fix it.